src/hex.cpp

```diff
@@ -73,28 +73,31 @@ namespace libtorrent
 	extern char const hex_chars[];
 
 	char const hex_chars[] = "0123456789abcdef";
+	static void to_hex(char const* in, size_t const len, char* out)
```

Comment:
``static`` in this context means internal linkage. C++ introduced a more uniform way of declaring internal linkage with anonymous namespaces.